src/matrix.jl

```diff
@@ -172,13 +172,53 @@ We exploit that
 to avoid taking the absolute (``|·|``).
 """
 function LinearAlgebra.opnorm(A::IntervalMatrix, p::Real=Inf)
-    if p == Inf || p == 1
-        return LinearAlgebra.opnorm(max.((-).(inf(A)), sup(A)), p)
+    if p == Inf
+        return _opnorm_inf(A)
+    elseif p == 1
+        return _opnorm_1(A)
     else
         error("the interval matrix norm for this value of p=$p is not implemented")
     end
 end
 
+# The operator norm in the infinity norm corresponds to the
+# maximum absolute value row-sum. Julia has column-major
+# ordering so for efficiency we iterate over the transpose of A.
```

Comment:
I doubt that this helps. `transpose` just creates a lazy wrapper, so it just complicates the logic. Did you try without `transpose`?

Reply:
I think that you mean version `_opnorm_inf_b` in the tests; https://nbviewer.jupyter.org/github/mforets/escritoire/blob/master/2020/Week7/Week7.ipynb . It is as efficient as the version i chose. I don't see a particular reason to choose one over the other. But sure, we can use version b.

Reply:
Yes, the fact that it is a lazy wrapper is what makes the code efficient.

Reply:
> I don't see a particular reason to choose one over the other.

I do: it is simpler to not do a `transpose`.